config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/string/EncodeDecodeUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.util.string;
+
+public class EncodeDecodeUtils {
+
+    private EncodeDecodeUtils() {
+    }
+
+    public static String encodePath(String pathToEncode) {
+        if (pathToEncode.contains("_¯") || pathToEncode.contains("¯_")) {
+            throw new IllegalArgumentException("Illegal characters in path. Cannot contain '_¯' or '¯_': " + pathToEncode);
+        }
+        pathToEncode = pathToEncode.replaceAll("/", "_¯");
```

Comment:
Would it be more intuitive to escape the characters besides replacing them? Since if I'm specifying policy in my yaml/json, I need to copy paste `_¯` or `¯_`

Ie: `folder\/withslash/policy\\test.xml` ->
* folder/withslash
  * policy\test

Reply:
I thought of that but the actual file that is saved will have the string `_¯` or `¯_` in it. This is mostly to cover the edge case of people putting a slash in file names because it was allowed by the policy manager. They shouldn't be doing that.